imports/plugins/core/orders/client/templates/workflow/shippingTracking.js

```diff
@@ -77,7 +77,7 @@ Template.coreOrderShippingTracking.events({
   "click [data-event-action=shipmentPacked]": () => {
     const template = Template.instance();
 
-    Meteor.call("orders/shipmentPacked", template.order, template.order.shipping[0], true);
+    Meteor.call("orders/shipmentPacked", template.order, template.order.shipping[0]);
```

Comment:
We can't assume that there is only one shipment for any given order. 

`template.order.shipping[0]` is too brittle for multiple shipments per order and in a marketplace setting we have to support multiple shipments per order.

In general, hard-coding the 0th index for an array is a code smell, especially as we continue to build multi-shop support